rrmngmnt/host.py

```diff
@@ -335,14 +353,61 @@ def is_connective(self, tcp_timeout=20.0):
         :return: True if host is connective, False otherwise
         :rtype: bool
         """
-        ret = False
         try:
-            self.executor().run_cmd(['true'], tcp_timeout=tcp_timeout)
-            ret = True
-        except (socket.error, socket.timeout):
-            ret = False
-        except Exception:
-            self.logger.warning("Unexpected exception", exc_info=True)
-            ret = False
-
-        return ret
+            self.logger.info(
+                "Check if host %s is connective via ssh", self.fqdn
+            )
+            self.run_command(['true'], tcp_timeout=tcp_timeout)
+            return True
+        except (socket.timeout, socket.error) as e:
+            self.logger.debug("Socket error: %s", e)
+        except paramiko.SSHException as e:
+            self.logger.debug("SSH exception: %s", e)
+        self.logger.error("Host %s does not connective via ssh", self.fqdn)
```

Comment:
what was decision process to change severity of this message?

in my opinion this line even shouldn't not be here. it is method 'is_connective' and does the job ... returns True/False according status of Host. without context of usecase you can not decide whether it is error or not ...

I would rather do not see it here.